creator-node/src/monitors/syncHistory.js

```diff
@@ -0,0 +1,73 @@
+const redisClient = require('../redis')
+const SyncHistoryAggregator = require('../snapbackSM/syncHistoryAggregator')
+
+// The window to determine the aggregate sync history count
+const ROLLING_WINDOW = 30 // days
+
+const getRollingSyncSuccessCount = async () => {
+  // Get the start and end dates of the rolling window
+  const today = new Date()
```

Comment:
all the Date() manipulation in this is kinda confusing since the api is not immediately obvious but also very slick 😎